spec/models/like_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { Like.new(created_at: Time.now, updated_at: Time.now, user_id: 1, post_id: 1) }
+  before { subject.save }
+
+  it 'User ID should be a number' do
+    subject.user_id = 'pizza'
+    expect(subject).to_not be_valid
+  end
+
+  it 'Post ID should be a number' do
+    subject.post_id = 'pasta'
+    expect(subject).to_not be_valid
```

Comment:
- [ ] There are 20 tests that are failing. Here you are using `user_id`, but you have to change this to `author_id`. Please fix it.

Reply:
resolved the `user_id` with `author_id`